packages/sdk-metrics/src/Meter.ts

```diff
@@ -50,6 +52,7 @@ export class Meter implements IMeter {
    * Create a {@link Histogram} instrument.
    */
   createHistogram(name: string, options?: MetricOptions): Histogram {
+    this._warnIfInvalidName(name);
```

Comment:
Is it possible to add the warning during the call to createInstrumentDescriptor?